packages/react/src/atoms/textArea/base.tsx

```diff
@@ -37,24 +39,37 @@ const TextAreaBase = (props: TextAreaBaseProps) => {
       "font",
       "padding",
       "margin",
+      "shadow",
     ],
     props,
     textAreaBaseClasses
   );
 
+  const labelClassNames = resolvedStyleProps(
+    "labelClasses",
+    ["margin"],
+    props,
+    textAreaBaseClasses
+  );
+
   return (
     <>
-      {props.label !== "" && <LabelBase {...props} />}
+      {props.label !== "" && (
+        <div className={labelClassNames}>
+          <LabelBase {...props} />
```

Comment:
@Subbu-betalectic This is also wrong, Please check Simple select input, we are passing labelBaseProps and we don't need to wrap this, please check TextInput